transfers/examples/pre-transfer/00_unbag.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# Script to re-package unzipped bags as standard transfers, utilizing checksums from bag manifest.
+# Assumes bags are structured as either bag/data/(content) or bag/data/objects/(content). 
+# Enables use of scripts to add metadata to SIP without failing transfer at bag validation.
+
+from __future__ import print_function, unicode_literals
+import os
+import shutil
+import sys
+
+def main(transfer_path):
+    transfer_path = os.path.abspath(transfer_path)
+
+    # check if transfer is an unzipped bag
+    if not os.path.isfile(os.path.join(transfer_path, 'bag-info.txt')):
+        return 1
+
+    # move files in data up one level if 'objects' folder already exists
+    data_path = os.path.join(transfer_path, 'data')
+    if os.path.isdir(os.path.join(data_path, 'objects')):
+        data_contents = os.listdir(data_path)
+        data_contents = [os.path.abspath(data_path) + '/' + filename for filename in data_contents]
```

Comment:
Would this be easier as `[os.path.join(data_path, filename) for filename in data_contents]`  `data_path` should already be absolute, since `transfer_path` is.